RecoDataProducts/inc/CosmicTrkFitFlag.hh

```diff
@@ -0,0 +1,26 @@
+#ifndef RecoDataProducts_CosmicTrkFitFlag_hh
+#define RecoDataProducts_CosmicTrkFitFlag_hh
+//Author: S Middleton
+//Purpose: Class to describe flag bits for Cosmic Track Fitting
+
+#include "GeneralUtilities/inc/BitMap.hh"
+#include <string>
+#include <map>
+namespace mu2e {
+
+  struct CosmicTrkFitFlagDetail {
```

Comment:
As we discussed the cosmic bit should be merged into TrkFitFlag.

Reply:
added 1 bit to TrkFitFlag for "isStraight" then can reuse other flags. This was discussed at the CM